ics.js

```diff
@@ -135,7 +135,7 @@ var ics = function(uidDomain, prodId) {
       // Since some calendars don't add 0 second events, we need to remove time if there is none...
       var start_time = '';
       var end_time = '';
-      if (start_hours + start_minutes + start_seconds + end_hours + end_minutes + end_seconds != 0) {
+      if (start_hours + start_minutes + start_seconds + end_hours + end_minutes + end_seconds !== 0) {
```

Comment:
That's wrong because the variables are strings. Your code will never enter the `if` clause. A (one possible) correct fix is: `!== "000000000000"`. I fixed it like this: https://github.com/michael-maltsev/ics.js/commit/f34a6ede128a514502a3d3046de32f7c22cde7ab

Reply:
@michael-maltsev Thanks for letting me know!